ballerina/tests/asb_admin_tests.bal

```diff
@@ -0,0 +1,1128 @@
+    deadLetteringOnFilterEvaluationExceptions: true,
+    forwardDeadLetteredMessagesTo: testQueue1,
+    forwardTo: testQueue1,
+    requiresSession: false,
+    lockDuration: lockDue,
+    maxDeliveryCount: 10,
+    status: ACTIVE,
+    userMetadata: userMetaData
+};
+UpdateSubscriptionOptions updateSubConfig = {
+    autoDeleteOnIdle: deletion,
+    defaultMessageTimeToLive: ttl,
+    deadLetteringOnMessageExpiration: true,
+    deadLetteringOnFilterEvaluationExceptions: true,
+    enableBatchedOperations: true,
+    forwardDeadLetteredMessagesTo: testQueue1,
+    forwardTo: testQueue1,
+    lockDuration: lockDue,
+    maxDeliveryCount: 10,
+    status: ACTIVE,
+    userMetadata: userMetaData
+};
+CreateQueueOptions queueConfig = {
+    autoDeleteOnIdle: deletion,
+    defaultMessageTimeToLive: ttl,
+    duplicateDetectionHistoryTimeWindow: dupdue,
+    forwardDeadLetteredMessagesTo: testQueue1,
+    forwardTo: testQueue1,
+    lockDuration: lockDue,
+    maxDeliveryCount: 10,
+    maxMessageSizeInKilobytes: 0,
+    maxSizeInMegabytes: 1024,
+    status: ACTIVE,
+    userMetadata: userMetaData,
+    enableBatchedOperations: true,
+    deadLetteringOnMessageExpiration: true,
+    requiresDuplicateDetection: false,
+    enablePartitioning: false,
+    requiresSession: false
+};
+UpdateQueueOptions updateQueueConfig = {
+    autoDeleteOnIdle: deletion,
+    defaultMessageTimeToLive: ttl,
+    duplicateDetectionHistoryTimeWindow: dupdue,
+    forwardDeadLetteredMessagesTo: testQueue2,
+    lockDuration: lockDue,
+    maxDeliveryCount: 10,
+    maxMessageSizeInKilobytes: 0,
+    maxSizeInMegabytes: 1024,
+    status: ACTIVE,
+    userMetadata: userMetaData,
+    enableBatchedOperations: true,
+    deadLetteringOnMessageExpiration: true,
+    requiresDuplicateDetection: false,
+    enablePartitioning: false,
+    requiresSession: false
+};
+CreateRuleOptions ruleConfig = {
+    rule: rule
+};
+UpdateRuleOptions updateRuleConfig = {
+    rule: rule
+};
+
+@test:Config {
+    groups: ["asb_admin"],
+    enable: true
+}
+function testCreateQueue() returns error? {
+    log:printInfo("[[testCreateQueue]]");
+    log:printInfo("Initializing Asb admin client.");
+    Administrator adminClient = check new (connectionString);
+    QueueProperties? queueProp = check adminClient->createQueue(testQueue1);
+    if queueProp is QueueProperties {
+        log:printInfo("Queue created successfully.");
+    } else {
+        test:assertFail("Queue creation failed.");
+    }
+}
+
+@test:Config {
+    groups: ["asb_admin"],
+    dependsOn: [testCreateQueue],
+    enable: true
+}
+function testQueueExists() returns error? {
+    log:printInfo("[[testQueueExists]]");
+    log:printInfo("Initializing Asb admin client.");
+    Administrator adminClient = check new (connectionString);
+    boolean? queueExists = check adminClient->queueExists(testQueue1);
+    if queueExists is boolean {
+        test:assertTrue(queueExists,msg = "Queue exists failed.");
```

Comment:
```suggestion
        test:assertTrue(queueExists, msg = "Queue exists failed.");
```

Reply:
Addressed in [af47f42](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/af47f42e025c870c173c07d46a984ed796dd7ab1)